### PR TITLE
feat: Vector search using python client

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

File: docs/concepts/vector-search/typescript/usage-guide.mdx [hunk 1]
```diff
@@ -1,11 +1,11 @@
+# Usage Guide
```
Comment (himank) [inline docs/concepts/vector-search/typescript/usage-guide.mdx:1]: Looks like there is a spelling mistake at line `58`, should be `await vectorDocStore.ensureIndex()`

File: docs/concepts/vector-search/python/index.mdx [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+      "size": 10
+    }
+  },
+  "grouped_hits": []
+}
+```
+
+</details>
+
+Here we see `reult.hits` is an array of all matching documents and some vector store metadata. The
```
Comment (himank) [inline docs/concepts/vector-search/python/index.mdx:197]: `result.hits`

Comment (adilansari) [reply docs/concepts/vector-search/python/index.mdx]: fixed

File: docs/concepts/vector-search/typescript/usage-guide.mdx (hunk 1 above)
Comment (adilansari) [reply docs/concepts/vector-search/typescript/usage-guide.mdx:1]: seems like typo in SDK

https://github.com/tigrisdata/tigris-vector-ts/blob/main/src/vectorstore.ts#LL38C22-L38C22

Comment (himank) [reply docs/concepts/vector-search/typescript/usage-guide.mdx:1]: hmm

**Review by himank — APPROVED** [review]

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**